test/unit/forge/ee/lib/sso/index_spec.js

```diff
@@ -365,5 +365,29 @@ d
             })
             ;(await app.db.models.TeamMember.getTeamMembership(app.user.id, teams.ATeam.id)).should.have.property('role', Roles.Owner)
         })
+        it('strip prefix and suffix from SAML groups', async function () {
+            // This should remove ownership from Alice in Team A
+
+            // Starting state:
+            // Alice owner ATeam
+
+            // Expected result:
+            // Alice owner ATeam - unchanged
+            await app.sso.updateTeamMembership({
+                'ff-roles': [
+                    'test_ff-ateam-magician_err',
+                    'test_ff-ateam-member_test',
+                    'test_ff-bteam-owner_test',
+                    'ff-ateam-admin_test'
+                ]
+            }, app.user, {
+                groupAssertionName: 'ff-roles',
+                groupAllTeams: true,
+                groupPrefixLength: 5,
+                groupSuffixLength: 5
```

Comment:
```suggestion
                    'test_ff-ateam-magician_err',
                    'test_ff-ateam-member_test2',
                    'test_ff-bteam-owner_test2',
                    'ff-ateam-admin_test'
                ]
            }, app.user, {
                groupAssertionName: 'ff-roles',
                groupAllTeams: true,
                groupPrefixLength: 5,
                groupSuffixLength: 6
```

Got itchy using the same value for both pre/suffix as it wouldn't spot any mix up in applying the wrong value to the wrong end.